sdk/src/main/java/com/karhoo/sdk/api/network/request/LoyaltyPreAuth.kt

```diff
@@ -0,0 +1,11 @@
+package com.karhoo.sdk.api.network.request
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class LoyaltyPreAuth(@SerializedName("currency") val currency: String? = null,
```

Comment:
the ios LoyaltyPreAuth contains 
`public let identifier: String`

I think the identifier can be removed in the iOS version (Android is fine?)